### DISCUSSION TITLE
更新日志 - 22.03.6 - 20240719

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#1712] [Announcements]
# 系统
1. 此版本之前的rk35xx系列的内核配置错误，docker从**默认路径**迁移到其他分区以后会导致容器和镜像丢失。
  此版本已修复此问题，但是如果之前没迁移docker数据，此次升级以后将会丢失已有的容器和镜像，需要重新部署容器（之前已迁移过的不受影响）。
  不过默认路径只有2GB空间，影响不大，如果确实需要保留之前的数据，可以在升级前使用docker的save和export命令分别导出镜像和容器数据，升级以后再导入，详情参考docker官方的文档。技术细节可查看此提交 https://github.com/istoreos/istoreos/commit/7194963cc2306a7ba9f1eca443c755d0c7a851fd

2. x86集成r8126驱动，以支持 Realtek 5G 以太网卡。其实之前固件中的r8125驱动已经支持此类网卡，只是需要在杂项设置中手动启用r8125驱动。此次更新以后r8125驱动将只支持2.5G网卡，r8126支持5G网卡并且默认启用。（rk35xx本就默认启用r8125驱动，r8126也会加上，也就是不受影响） #1700

# 应用
* QuickStart：
  1. 前端初步完成国际化（大部分是AI翻译的），如果Luci当前不是中文，quickstart界面将显示英文
  2. 修复RAID创建失败问题

**Comment by xifeng181** [comment]
e20c之前是迁移过docker路径的，这次升级以后，docker无法启用了。

**Reply by jjm2473** [reply]
无法启用跟这个变更无关。这个变更只会丢失容器和镜像（识别不到），其他都是正常的，拉镜像或者部署都没问题。

你自己看看系统日志到底为什么docker启动失败吧。

**Reply by hurcaguari** [reply]
我的也有个容器没有正常运行重启就好了
